deployment/ansible/deploy.yml

```diff
@@ -5,6 +5,7 @@
 
   vars:
     project_root: "{{ ansible_env.HOME }}/node"
+    chain_db_path: "/.chain"
```

Comment:
@dincho Do I understand correctly that once the default path moves to `.` you need neither this line nor L45-L67?

I would suggest not adding this workaround, but rather ensure the related PRs get merged...

Reply:
Was it really meant to be a hidden directory? Why?